content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -46,6 +46,24 @@ Our current process is as follows:
 
 - If one or more issues that were planned for an iteration are looking to **not get finished** (which includes testing) in the [current iteration](https://github.com/orgs/sourcegraph/projects/200/views/1) (while maintaining sustainable work practices) the assignee **raises this as soon as possible asynchronously** to the team (including the PM and EM), e.g. on the GitHub issue or Slack. These issues then **become _proposed_ issues for the next iteration** (meaning nothing carries over automatically, but we also don't just drop and forget missed issues).
 
+## How we think about issue sizes
+
+When sizing an issue please keep in mind the size includes the entire lifecycle of the issue. The coding, any testing both automated and manual, as well as PR reviews, documentation, and any follow-up needed.
+
+When thinking about the size of an issue we try to stay away from actual time estimates. But generally, 1 and 2 sizes would probably finish in less than a full workday. 3 and 5 probably no more than 2 workdays. 8 and 13 get closer to a full week and beyond that begins to span whole weeks or even months.
```

Comment:
I'm a little uncomfortable with this, it feels very contradictory. If I were reading this for the first time, I'm not sure what to think - the first line says not to think in terms of time, but the only frame of reference I'm being given is time.

It might be useful to instead reframe in terms of something else?
```suggestion
When thinking about the size of an issue we try to stay away from actual time estimates. Generally, 1 and 2 sizes should be considered small. 3 and 5 are medium issues. 8 and 13 can be considered large issues. Any number larger than 13 can be considered significantly large, and possibly could represent a small project.
```